saleor/graphql/account/schema.py

```diff
@@ -164,9 +165,9 @@ def resolve_me(self, info):
     def resolve_staff_users(self, info, query=None, **_kwargs):
         return resolve_staff_users(info, query=query)
 
-    @permission_required("account.manage_users")
+    @one_of_permissions_required(["account.manage_staff", "account.manage_users"])
```

Comment:
I don't think we need to check twice the permissions? Through this wrap and then we check again in `resolve_user`

Reply:
We need this because if an anonymous user tries to get this query that can cause different error than permission denied.